idaes/core/util/testing.py

```diff
@@ -309,6 +310,9 @@ def build(self):
         # Attribute to switch flow basis for testing
         self.basis_switch = 1
 
+        self.set_default_scaling("reaction_rate", 101, 'r1')
```

Comment:
The reason for this order is that the index is optional.  But I agree.  Style-wise, it may be better to use named arguments and flip it. ```self.set_default_scaling("raction_rate", index="r1", value=101)```